charts/enterprise/blocky/docs/setup-guide.md

```diff
@@ -59,10 +76,18 @@ certain blocklists if you find legitimate traffic being blocked.
 
 1. Pick a Group Name for your blocklists.
 2. Add List entries for each blocklist by URL.
-  ![blocky-blacklist](./img/blocky-blacklist.png)
+   ![blocky-blacklist](./img/blocky-blacklist.png)
 3. Add a Clients Group Block and set Client Group Name to `default`
 4. Under Groups Entry enter the Group name you used above.
-  ![blocky-blacklist-group](./img/blocky-blacklist-group.png)
+   ![blocky-blacklist-group](./img/blocky-blacklist-group.png)
+
+## Networking
+
+If you want to use Blocky on your local network to take advantage of the filtering above, or the k8s-gateway below, you 
+need to ensure your loadbalancer IP address is set and won't change.
+
+You need to configure the systems on your network to use the loadbalancer IP address configured above.  This will either be done per system (seting a `static IP` for your Network Interface and having the apps use it)
```

Comment:
```suggestion
You need to configure the devices on your network to use the static IP address configured above.  This can be done per device or by changing the DHCP or DNS settings on your router.
```